nautobot/core/settings.py

```diff
@@ -833,3 +833,5 @@ def UI_RACK_VIEW_TRUNCATE_FUNCTION(device_display_name):
     :type: str
     """
     return str(device_display_name).split(".")[0]
+
+HIDE_RESTRICTED_UI = True
```

Comment:
Was this from testing? It's already a Constance config setting which should be retreived with `settings_or_config`.

Reply:
Yeah definitely should revert this.